.github/workflows/devcontainer.yml

```diff
@@ -0,0 +1,23 @@
+name: Devcontainer
```

Comment:
Commenting here to prevent accidental merges. I need to mark this PR "Ready for review" to test the workflow... 🙃